packages/dds/tree/src/shared-tree-core/editManagerFormatV1toV4.ts

```diff
@@ -21,17 +22,17 @@ const noAdditionalProps: ObjectOptions = { additionalProperties: false };
 export interface EncodedEditManager<TChangeset> {
 	readonly trunk: readonly Readonly<SequencedCommit<TChangeset>>[];
 	readonly branches: readonly [SessionId, Readonly<EncodedSummarySessionBranch<TChangeset>>][];
-	readonly version: 1 | 2 | 3 | 4;
+	readonly version: EditManagerFormatVersion;
```

Comment:
With the object as const instead of enum pattern, I wasn't able to set the type to only accept values 1 to 4 here other than using 1 | 2 | 3 | 4.

Reply:
```suggestion
	readonly version:
		| typeof EditManagerFormatVersion.v1
		| typeof EditManagerFormatVersion.v2
		| typeof EditManagerFormatVersion.v3
		| typeof EditManagerFormatVersion.v4;
```

Reply:
This makes the version's type to just be a number. It doesn't limit the values to be 1 to 4. I am guessing that was the intention of the code before this change.

Reply:
That's because of the missing `as const` :)